app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+
+  def is_loggedin
+    redirect_to root_path, notice: "User Not Logged In" if current_user.blank?
+  end
+
+  def current_user
+    User.find_by(id: session[:user_id])
```

Comment:
This will run a database query each time we call this method. Can we minimize these queries?